src/shared/codelens/codeLensUtils.ts

```diff
@@ -44,7 +44,9 @@ export async function makeCodeLenses({
     const workspaceFolder: vscode.WorkspaceFolder | undefined = vscode.workspace.getWorkspaceFolder(document.uri)
 
     if (!workspaceFolder) {
-        throw new Error(`Source file ${document.uri} is external to the current workspace.`)
+        getLogger().error(`Source file ${document.uri} is external to the current workspace.`)
```

Comment:
since we don't have a stacktrace now a prefix is particularly important
```suggestion
        getLogger().error(`makeCodeLenses: source file is external to workspace: ${document.uri}`)
```